apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -257,6 +257,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
   blocklyWrapper.Block.prototype.appendEndRowInput =
     blocklyWrapper.Block.prototype.appendDummyInput;
 
+  blocklyWrapper.Block.prototype.setStyle = function () {};
```

Comment:
Is it worth leaving a comment to the effect of the note in your PR description?

> Block.setStyle doesn't normally exist on CDO Blockly, so I patched in the wrapper so that it doesn't cause any errors when called.